From 363d49d73c5bf474786ad09e2b86884e520b9bfe Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Wed, 4 Nov 2009 20:40:57 +0100
Subject: [PATCH 3/5] gta02_defconfig: update config

* enable MFD_GLAMO_MCI for booting from SD
* enable touchscreen driver
* enable touchscreen filters
* enable /proc/config.gz
* drop gta01 support
* enable wm8753 (Kconfig option renamed)
* enable S3C24XX_ADC for touchscreen
---
 arch/arm/configs/gta02_defconfig |   20 +++++++++++++++-----
 1 files changed, 15 insertions(+), 5 deletions(-)

diff --git a/arch/arm/configs/gta02_defconfig b/arch/arm/configs/gta02_defconfig
index efc6d94..b0a6d72 100644
--- a/arch/arm/configs/gta02_defconfig
+++ b/arch/arm/configs/gta02_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.31
-# Fri Oct  9 11:35:24 2009
+# Sun Nov  8 11:59:38 2009
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -50,7 +50,8 @@ CONFIG_CLASSIC_RCU=y
 # CONFIG_PREEMPT_RCU is not set
 # CONFIG_TREE_RCU_TRACE is not set
 # CONFIG_PREEMPT_RCU_TRACE is not set
-# CONFIG_IKCONFIG is not set
+CONFIG_IKCONFIG=m
+CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_BUF_SHIFT=18
 # CONFIG_GROUP_SCHED is not set
 # CONFIG_CGROUPS is not set
@@ -191,7 +192,7 @@ CONFIG_S3C24XX_GPIO_EXTRA=64
 CONFIG_S3C24XX_GPIO_EXTRA64=y
 CONFIG_S3C2410_DMA=y
 # CONFIG_S3C2410_DMA_DEBUG is not set
-# CONFIG_S3C24XX_ADC is not set
+CONFIG_S3C24XX_ADC=y
 CONFIG_PLAT_S3C=y
 CONFIG_CPU_LLSERIAL_S3C2440_ONLY=y
 CONFIG_CPU_LLSERIAL_S3C2440=y
@@ -233,6 +234,7 @@ CONFIG_S3C2410_GPIO=y
 # CONFIG_MACH_TCT_HAMMER is not set
 # CONFIG_MACH_VR1000 is not set
 # CONFIG_MACH_QT2410 is not set
+# CONFIG_MACH_NEO1973_GTA01 is not set
 
 #
 # S3C2412 Machines
@@ -746,6 +748,13 @@ CONFIG_KEYBOARD_GPIO=y
 # CONFIG_INPUT_JOYSTICK is not set
 # CONFIG_INPUT_TABLET is not set
 CONFIG_INPUT_TOUCHSCREEN=y
+CONFIG_TOUCHSCREEN_FILTER=y
+CONFIG_TOUCHSCREEN_FILTER_GROUP=y
+CONFIG_TOUCHSCREEN_FILTER_MEDIAN=y
+CONFIG_TOUCHSCREEN_FILTER_MEAN=y
+CONFIG_TOUCHSCREEN_FILTER_LINEAR=y
+CONFIG_TOUCHSCREEN_S3C2410=y
+# CONFIG_TOUCHSCREEN_S3C2410_DEBUG is not set
 # CONFIG_TOUCHSCREEN_ADS7846 is not set
 # CONFIG_TOUCHSCREEN_AD7877 is not set
 # CONFIG_TOUCHSCREEN_AD7879_I2C is not set
@@ -965,10 +974,11 @@ CONFIG_PCF50633_ADC=y
 CONFIG_PCF50633_GPIO=y
 # CONFIG_AB3100_CORE is not set
 # CONFIG_EZX_PCAP is not set
+# CONFIG_MFD_PCF50606 is not set
 CONFIG_MFD_GLAMO=y
 CONFIG_MFD_GLAMO_FB=y
 CONFIG_MFD_GLAMO_GPIO=y
-# CONFIG_MFD_GLAMO_MCI is not set
+CONFIG_MFD_GLAMO_MCI=y
 # CONFIG_MEDIA_SUPPORT is not set
 
 #
@@ -1076,7 +1086,7 @@ CONFIG_SND_SEQUENCER_OSS=y
 CONFIG_SND_SOC=m
 CONFIG_SND_S3C24XX_SOC=m
 CONFIG_SND_S3C24XX_SOC_I2S=m
-CONFIG_SND_S3C24XX_SOC_NEO1973_GTA02_WM8753=m
+CONFIG_SND_S3C24XX_SOC_NEO1973_WM8753=m
 # CONFIG_SND_S3C24XX_SOC_LN2440SBC_ALC650 is not set
 # CONFIG_SND_S3C24XX_SOC_S3C24XX_UDA134X is not set
 CONFIG_SND_SOC_I2C_AND_SPI=m
-- 
1.6.5.2

